Formula/vis.rb

```diff
@@ -0,0 +1,44 @@
+class Vis < Formula
+  desc "Vim-like text editor"
+  homepage "https://github.com/martanne/vis"
+  url "https://github.com/martanne/vis/archive/v0.2.tar.gz"
+  sha256 "3e5b81d760849c56ee378421e9ba0f653c641bf78e7594f71d85357be99a752d"
+
+  head "https://github.com/martanne/vis.git"
+
+  keg_only :shadowed_by_osx, <<-EOS.undent
+    This package conflicts with the OS X system utility /usr/bin/vis.
+    Force-linking is not recommended as it may cause problems with other
+    programs which rely on this utility. Instead, a wrapper script
+    `vise` ("vis editor") has been provided to set necessary environment
+    and run the editor. Invoke it by running
+
+        $ `brew --prefix vis`/bin/vise
+
+    If you plan on doing this often, you may wish to add an alias
+    to your shell startup file:
+
+        alias vise=$(brew --prefix vis)/bin/vise
```

Comment:
If there's a wrapper script you could perhaps just make `libexec` the prefix, add the wrapper script and not make it keg-only?